dev/com.ibm.ws.jpa.tests.jpa_31_fat/test-applications/jpa31/src/io/openliberty/jpa/tests/jpa31/web/TestNewQueryTimeFunctionsServlet.java

```diff
@@ -172,9 +172,9 @@ public void testLocalDateFunction_JPQL() throws Exception {
     @Test
     public void testLocalTimeFunction_JPQL() throws Exception {
         // TODO: Derby needs to switch from TIMESTAMP to TIME for this to work right; https://github.com/eclipse-ee4j/eclipselink/issues/1544
-        if (isDerby())
-            return;
-
+//        if (isDerby())
+//            return;
+//
```

Comment:
Should these comments be removed now that the issues/TODOs are resolved?

Reply:
Probably should, yeah.

Reply:
fixed